spec.bs

```diff
@@ -2837,9 +2840,6 @@ content in the <{fencedframe}> or its embedder, exactly one of two things will h
   :: |automaticBeaconsAllowed|.
 </div>
 
-<span class=XXX>TODO: Call the [=fenced frame config instance/on navigate callback=] at the
```

Comment:
The `on navigate callback` should be called when the urn/config is loaded into a fenced frame, not when there is a top-level navigation.

Reply:
Fixed.  I misinterpreted Fenced_Frames_Ads_Reporting.md to mean that `reserved.top_navigation_start` is sent on initial ad navigation.